Formula/ehco.rb

```diff
@@ -0,0 +1,42 @@
+class Ehco < Formula
+  desc "Network relay tool and a typo :)"
+  homepage "https://github.com/Ehco1996/ehco"
+  url "https://github.com/Ehco1996/ehco/archive/refs/tags/v1.0.6.tar.gz"
+  sha256 "326c468c3790ad01031e52ccb4efcfa5e331d2198fdb13137749c67e8eaacf38"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "cmd/ehco/main.go"
+  end
+
+  test do
+    version_info = shell_output("#{bin}/ehco -v")
+    assert_match(/ehco version */, version_info)
+
+    help_info = shell_output("#{bin}/ehco -h > help.txt")
+    assert_predicate testpath / "help.txt", :exist?
```

Comment:
Why do we need to print this to a file? If we do, use the `write` method instead of shell redirection. I'd rather we didn't print to a file at all, though.

Reply:
yes, don't need to write to file ,and i just delete this part of test.